libs/core/src/scss/base/_functions.scss

```diff
@@ -226,7 +226,7 @@
 @mixin trailing-icon($svg: 'link') {
   $icon-scaling-factor: 1.5em;
 
-  @if str-index($svg, 'http') {
+  @if string.index($svg, 'http') {
```

Comment:
The package-lock file was updated after bumping, and suddenly I see stylelint errors. 
I can see from the package-lock file that our stylelint dep was bumped from 16.3.1 -> 16.5.0, so that might be why these two errors suddenly emerged? 🤷

Reply:
Yep, thats it. 

From the stylelint/scss changelog: Fixed: no-global-function-names report errors inside at-rules (https://github.com/stylelint-scss/stylelint-scss/pull/988).